src/ENSMetadata.sol

```diff
@@ -61,11 +61,6 @@ contract ENSMetadata {
     }
 
     function verifyENS() public returns (bool) {
```

Comment:
I think we are not calling this method internally in our contract, in that case can we make it `external` insetead of `public`

Reply:
will update it right now